docs/commands/clusters.adoc

```diff
@@ -65,15 +65,31 @@ This will create a single node cluster, running the index, data query and search
 The full list of flags supported by `clusters create` is:
 
 ```
-  --name <String> - the name of the cluster
-  --provider <String> - the cloud provider
-  --version <String> - the couchbase server version
-  --capella <String> - the Capella organization to use
-  --nodes <Int> - the number of nodes in the cluster
-  --project <String> - the Capella project to use
+  --name <string>: the name of the cluster
+  --provider <string>: the cloud provider
+  --version <string>: the couchbase server version
+  --capella <string>: the Capella organization to use
+  --nodes <int>: the number of nodes in the cluster
+  --project <string>: the Capella project to use
+  --free-tier: deploy a free tier cluster
```

Comment:
What other types of cluster deployment are there? It seems more like there should be a `plan` or something option to cover additional tiers maybe?